Exiled.API/Features/Core/Components/TickComponent.cs

```diff
@@ -24,7 +24,7 @@ public sealed class TickComponent : EObject
         /// The default fixed tick rate (60 per second).
         /// </summary>
 #pragma warning disable SA1310
-        public const float DEFAULT_FIXED_TICK_RATE = 0.016f;
+        public const float DEFAULT_FIXED_TICK_RATE = 1f / 60f;
```

Comment:
While this produces the same result, it just produces roughly the same result. As seen below, the difference is miniscule. If you feel that this works better, it would need to be tested and monitored in a controlled environment for an extended period of time.

![image](https://github.com/user-attachments/assets/33ed89e2-6b85-49d1-9288-1e60250f4d60)

Reply:
For spelling errors: I did not have a suitable spell checker on the computer where I wrote this code and is not my mothertongue.

Roslyne can do math optimisation and avoid floating-point error.
In my country floating-point error are see durring the Grade 12 or first year when pushing Bachelor's degree...
0.016f * 60f result in 0.96;
(1f / 60f) * 60f result in 1.